### PR TITLE
Add REUSE_CERT to skip creaing new ca/cert files

### PR DESCRIPTION
**Pull request by xychu**
**What this PR does / why we need it**:
It's maybe useful with `PERSERVE_ETCD=true` to run `local-up-cluster.sh` using existed
certs, so that service account secrets will keep valid.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -514,37 +550,11 @@ function start_apiserver {
...
-    # Create CA signers
-    if [[ "${ENABLE_SINGLE_CA_SIGNER:-}" = true ]]; then
-        kube::util::create_signing_certkey "${CONTROLPLANE_SUDO}" "${CERT_DIR}" server '"client auth","server auth"'
-        sudo cp "${CERT_DIR}/server-ca.key" "${CERT_DIR}/client-ca.key"
-        sudo cp "${CERT_DIR}/server-ca.crt" "${CERT_DIR}/client-ca.crt"
-        sudo cp "${CERT_DIR}/server-ca-config.json" "${CERT_DIR}/client-ca-config.json"
-    else
-        kube::util::create_signing_certkey "${CONTROLPLANE_SUDO}" "${CERT_DIR}" server '"server auth"'
-        kube::util::create_signing_certkey "${CONTROLPLANE_SUDO}" "${CERT_DIR}" client '"client auth"'
+    if [[ "${REUSE_CERTS}" == "false" ]]; then
```
Comment (fejta) [inline hack/local-up-cluster.sh]: if REUSE_CERTS != "true" might be more appropriate?

File: hack/local-up-cluster.sh [hunk 2]
```diff
@@ -231,6 +231,9 @@ ROOT_CA_FILE=${CERT_DIR}/server-ca.crt
 ROOT_CA_KEY=${CERT_DIR}/server-ca.key
 CLUSTER_SIGNING_CERT_FILE=${CLUSTER_SIGNING_CERT_FILE:-"${ROOT_CA_FILE}"}
 CLUSTER_SIGNING_KEY_FILE=${CLUSTER_SIGNING_KEY_FILE:-"${ROOT_CA_KEY}"}
+# Reuse certs will skip generate new ca/cert files under CERT_DIR
+# it's useful with PRESERVE_ETCD=true because new ca will make existed service account secrets invalided
+REUSE_CERTS=${REUSE_CERTS:-"false"}
```
Comment (fejta) [inline hack/local-up-cluster.sh]: FYI: we have had issues with people confusing "false" with `-z`

**Review by fejta — COMMENTED** [review]
/ok-to-test
@kubernetes/sig-cluster-lifecycle-pr-reviews 

What is the status of local-cluster-up? Do we want people using this? Do we have an alternative ready?

**Comment by xychu** [comment]
@fejta thanks for your review, I've updated :-)

**Review by fejta — APPROVED** [review]
/lgtm

**Pull request merged**